Python/Product/PythonTools/PythonTools/LanguageServerClient/StreamHacking/StreamIntercepter.cs

```diff
@@ -52,8 +52,16 @@ class StreamIntercepter : Stream {
         public override long Seek(long offset, SeekOrigin origin) => baseStream.Seek(offset, origin);
         public override void SetLength(long value) => baseStream.SetLength(value);
         public override void Write(byte[] buffer, int offset, int count) {
-            var writeHandlerResult = writeHandler.Invoke(new StreamData{ bytes = buffer, offset = offset, count = count });
-            baseStream.Write(writeHandlerResult.bytes, writeHandlerResult.offset, writeHandlerResult.count);
+            if (writeHandler != null) {
```

Comment:
Refactored the stream intercepting to skip intercepting if already handled the appropriate message. This speeds up intellisense a little bit as we don't deserialize every message twice (once in our stuff and once in the VS SDK bits)